common/build/eslint-config-fluid/package.json

```diff
@@ -61,5 +62,10 @@
 		"sort-json": "^2.0.1",
 		"typescript": "~5.1.6"
 	},
+	"dependenciesMeta": {
```

Comment:
Why is this needed?

Reply:
https://pnpm.io/package_json#dependenciesmetainjected

I added the field to ensure the linking between the nested packages. 

When working on it, I had also seen a suggestion from the creators of PNPM to use `dependenciesmetainjected` to ensure `pnpm i` correctly installing packages (or linking) in `node_modules`, but can't find the response at this moment. 

I can remove the field and try to verify whether everything still works or not.

Reply:
It's working without the `dependenciesMeta` field so I have removed it.